content/operate/rc/databases/rdi/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+Title: Data Integration
+alwaysopen: false
+categories:
+- docs
+- operate
+- rc
+description: Use Redis Data Integration with Redis Cloud.
+hideListLinks: true
+weight: 99
+---
+
+Redis Cloud now supports [Redis Data Integration (RDI)]({{<relref "integrate/redis-data-integration">}}), a fast and simple way to bring your data into Redis.
+
+The applications that write data to your primary database are not always the same application that need faster access to the data. Redis Cloud uses [ingest]({{<relref "/integrate/redis-data-integration/ingest/">}}) to help you offload all read queries from the application database to Redis.
```

Comment:
What you've said is correct but RDI works even if it's basically always the same app doing reads and writes.
```suggestion
A relational database usually handles queries much more slowly than a Redis database. If your application uses a relational database and makes many more reads than writes (which is the typical case) then you can improve performance by using Redis as a cache to handle the read queries quickly. Redis Cloud uses [ingest]({{<relref "/integrate/redis-data-integration/ingest/">}}) to help you offload all read queries from the application database to Redis automatically.
```